beetsplug/xtractor/__init__.py

```diff
@@ -7,12 +7,47 @@
 import os
 
 from beets.plugins import BeetsPlugin
+from beets.dbcore import types
 from confuse import ConfigSource, load_yaml
 from beetsplug.xtractor.command import XtractorCommand
 
 
 class XtractorPlugin(BeetsPlugin):
     _default_plugin_config_file_name_ = 'config_default.yml'
+    item_types = {
+        'average_loudness': types.Float(6),
+        'beats_count': types.INTEGER,
+        # 'chords_changes_rate': types.Float(6),
+        # 'chords_key': types.STRING,
+        # 'chords_number_rate': types.Float(6),
+        # 'chords_scale': types.STRING,
+        # 'key_strength': types.Float(6),
+        'danceable': types.Float(6),
+        'danceability': types.Float(6),
+        'gender': types.STRING,
+        'is_male': types.Float(6),
```

Comment:
I am not well versed with this plugin, but wanted to double check that this is not `boolean`?

Reply:
Yes definitely. That was my plan actually! Thanks for reviewing appreciate it!

Reply:
Whoops, I was wrong again: I wanted to doublecheck whether this should be boolean, forgot about it, then you came in....

Seems like this really is a numerical field that gives a "percentage" of how male or female a tune/voice is sounding :-)

```
xtractor: Audiodata: {'bpm': 84, 'danceability': 1.19403266907, 'beats_count': 629, 'average_loudness': 0.614515542984, 'danceable': 0.89466881752, 'gender': 'female', 'is_male': 0.0629677921534, 'is_female': 0.937032222748, 'genre_rosamerica': 'rhy', 'voice_instrumental': 'voice', 'is_voice': 0.761259257793, 'is_instrumental': 0.238740727305, 'mood_acoustic': 0.138670727611, 'mood_aggressive': 0.0114292930812, 'mood_electronic': 0.713214933872, 'mood_happy': 0.110149629414, 'mood_sad': 0.723113715649, 'mood_party': 0.423510432243, 'mood_relaxed': 0.826796770096, 'mood_mirex': 'Cluster3', 'mood_mirex_cluster_1': 0.0496428199112, 'mood_mirex_cluster_2': 0.265290796757, 'mood_mirex_cluster_3': 0.306235015392, 'mood_mirex_cluster_4': 0.282513767481, 'mood_mirex_cluster_5': 0.0963176339865}
```